client/src/types/filters.ts

```diff
@@ -0,0 +1,5 @@
+export type Filters = {
+  minPrice: number;
+  maxPrice: number;
+  fees: boolean | null;
+};
```

Comment:
Questa cartella è più pensata per tipi di dominio, non per qualsiasi tipo in generale. È meglio dichiarare i tipi locali a dove più logicamente dovrebbero stare, non ha senso farlo in una cartella separate per cazzi suoi rendendo solo la ricerca più difficile. Per i tipi di dominio ha senso avere una loro cartella perché non fanno parte di nessuna area nello specifico, soprattutto nel nostro caso che abbiamo solo un'app e non diverse sezioni, altrimenti avremmo domini per feature